source/intro.tex

```diff
@@ -89,7 +89,7 @@
 read or modify the value of an object
 
 \begin{defnote}
-Only glvalues of scalar type can be used to access objects.
+Only glvalues of scalar type (\iref{basic.types.general}) can be used to access objects.
```

Comment:
Current usages of `\iref` are not parenthesized. I _guess_ additional `()` will make the citation doubly parenthesized.
```suggestion
Only glvalues of scalar type\iref{basic.types.general} can be used to access objects.
```